Managers/MovableMan.h

```diff
@@ -942,13 +942,13 @@ class MovableMan : public Singleton<MovableMan>, public Serializable {
     // A map to give a unique contiguous identifier per-actor. This is re-created per frame.
     std::unordered_map<const Actor *, int> m_ContiguousActorIDs;
     // List of items that are pickup-able by actors
-    std::deque<MovableObject *> m_Items;
+	std::deque<HeldDevice *> m_Items;
```

Comment:
Should probably have a changelog entry for this so people know about it.
Also, causeless mentioned issues. I don't agree with his other points, but if it causes erge conflicts with the already difficult to work with ai stuff, that'll suck, so it may be good to hold off on this for a separate quick pr after that.

Reply:
I can quite easily imagine in the future wanting to spawn in a random object that isn’t a HeldDevice or Particle. Like, one day I decide to implement stacks of boxes, or let some actors limbs be shot off of them without being destroyed, or whatever. I feel like changing this is then either going to require changing that back, or adding a new list and bunch of extra handling code on MovableMan for no benefit.

Reply:
Fair enough, but imo
1. If it has to be changed back, it's not a huge deal. This change provides a concrete benefit for lua scripts (not having to cast everything, which is both a hassle and requires more lua -> cpp boundary crossing) and I guess other stuff that uses it. Right now basically all of the code assumes MovableMan.Items holds HeldDevices, and this is just formalizing it.
2. I'm not sure if there's much need for a stack of boxes to be some other thing that's an item but not a HeldDevice, but I'm shot off limbs are already, and should be, in the particles collection, since they can't be picked up or interacted with like items, and particles is the catch-all group. The items collection basically exists for "stuff you can pick up", which currently is always some child of HeldDevice. If you want to have non-device pickupables then the object hierarchy needs changing anyway - some stuff will have to be pulled out of HeldDevice and moved over, and this can be updated to account for that.

Reply:
That's fair enough. It's not a huge issue in any case, I'm probably overthinking